src/lora.c

```diff
@@ -6,6 +6,7 @@ static twr_cmwx1zzabz_t modem;
 
 lora_state_t lora;
 
+static bool last_uplink_confirmed = false; 
```

Comment:
No need to explicitly initialize this variable to false. The C compiler will do that for you.